pages/[user]/[type].tsx

```diff
@@ -1,22 +1,23 @@
-import { useEffect, useState } from "react";
-import { GetServerSideProps, GetServerSidePropsContext } from "next";
-import Head from "next/head";
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
This eslint rule is no good as it prevents you from using type _inference_ which is like the best feature of TypeScript and the whole premise of [tRPC](https://trpc.io)

Reply:
See #462